doc/fparser2.rst

```diff
@@ -102,8 +102,8 @@ executing the AST. For example:
     >>> reader = FortranFileReader("compute_unew_mod.f90",
                                    ignore_comments=False)
     >>> f2008_parser = ParserFactory().create(std="f2008")
-    >>> ast = f2008_parser(reader)
-    >>> print ast
+    >>> parse_tree = f2008_parser(reader)
+    >>> print parse_tree
```

Comment:
This example continues to use the old `ast` variable at line 119. The text after this example continues to talk about an AST when it should be a parse tree.

Reply:
I've removed the use of ast apart from the name of the walk  function which is still walk_ast.